tests/stc_containers/leaks_check.py

```diff
@@ -0,0 +1,62 @@
+
+# pylint: disable=missing-function-docstring, missing-module-docstring
+def create_list():
```

Comment:
```suggestion
# pylint: disable=missing-function-docstring, missing-module-docstring, unused-variable

def create_list():
```

Reply:
The change in the bot file doesn't make the pylint test pass. It just stops this change from causing an error in the pyccel_lint test

Reply:
Codacy also runs pylint and doesn't use the bot